rest-api/offline/biobank_samples_pipeline.py

```diff
@@ -183,6 +191,7 @@ def _create_sample_from_row(row, biobank_id_prefix):
   sample = BiobankStoredSample(
       biobankStoredSampleId=row[_Columns.SAMPLE_ID],
       biobankId=biobank_id,
+      biobankOrderIdentifier=row[_Columns.BIOBANK_ORDER_IDENTIFIER],
```

Comment:
We should add some validation logic above that either bails or returns None if the order ID is not set... we shouldn't import samples without order IDs moving forward.

Reply:
@danqrodney Just to be clear: by order_id you mean biobank_ordered_sample.order_id ?

Reply:
No, I mean this field -- biobank_stored_sample.biobank_order_identifier (after it's renamed).

Reply:
done.